python/cuvs/cuvs/test/test_distance.py

```diff
@@ -21,6 +21,7 @@
 from cuvs.distance import pairwise_distance
 
 
+@pytest.mark.parametrize("times", range(20))
```

Comment:
Whats this times parameter used for? I don't see it used in the test it self - 

Are you just trying to run this test multiple times here to stress test it?

Reply:
Yeah, it's just for testing multiple times and to guarantee the reproducing on one going because the possibility is close to ~10% empirically.